aten/src/ATen/core/aten_interned_strings.h

```diff
@@ -507,6 +507,8 @@ _(aten, narrow) \
 _(aten, narrow_copy) \
 _(aten, native_batch_norm) \
 _(aten, native_batch_norm_backward) \
+_(aten, native_layer_norm) \
```

Comment:
We probably don't need duplicated symbols.
:facepalm: I see another rebuild to verify it tho

Reply:
Duplicated symbols? They were used in shape_inference.cpp.

Reply:
Oops, sorry I read the `native_batch_norm` two lines above as `native_layer_norm`. It's fine, we can keep them in.

On a side note, if you just need to identify operator kind symbol `aten::native_layer_norm`, you don't need to define them as interned string here. Operators from aten are registered dynamically when you load the library.
You cannot use `aten::native_layer_norm` without defining them as interned string, but you can WAR with `c10::Symbol::fromQualString("aten::native_layer_norm")`.

Reply:
`aten::native_layer_norm` is used in a switch statement, which expects a `constexpr`. `c10::Symbol::fromQualString("aten::native_layer_norm")` isn't a `constexpr `so it doesn't work.